src/vs/workbench/contrib/debug/browser/debugAdapterManager.ts

```diff
@@ -157,6 +157,11 @@ export class AdapterManager extends Disposable implements IAdapterManager {
 					},
 					'presentation': presentationSchema,
 					'internalConsoleOptions': INTERNAL_CONSOLE_OPTIONS_SCHEMA,
+					'multipleSessionWarning': {
+						type: 'boolean',
+						description: nls.localize('multipleSessionWarning', "Show a warning dialog when starting the same debug session."),
```

Comment:
"another instance of the same debug session"

Reply:
A "debug session" is the result of starting a "debug configuration" (or starting "debug").
It is not possible to start "the same debug session" or "another instance of the same debug session".

My proposal:
"Warn when trying to start the same debug configuration more than once."